signbank/dictionary/templates/dictionary/public_gloss_list.html

```diff
@@ -181,7 +181,7 @@ <h5><strong>{% blocktrans %}Copyright and license:{% endblocktrans %}</strong></
             {% endif %}
             </div>
             <div class="panel-footer">
-            {% for glosstranslations in obj.glosstranslations_set.all %}
+            {% for glosstranslations in obj.glosstranslations_set.all reversed %}
```

Comment:
Same here

Reply:
I'm going to delete this pull request and re-submit it, as I think there is possibly a better approach at the database level.